HelpMyStreetFE/HelpMyStreetFE/Services/Groups/GroupService.cs

```diff
@@ -159,6 +159,10 @@ public async Task<List<Group>> GetChildGroups(int groupId)
         public async Task<List<SupportActivityDetail>> GetSupportActivitesForRegistrationForm(RegistrationFormVariant registrationFormVariant)
         {
             var response = await _groupRepository.GetRegistrationFormSupportActivies(registrationFormVariant);
+            if (response.SupportActivityDetails.Count() == 0)
+            {
+                throw new Exception("No support activies for registration form type");
```

Comment:
Minor improvement: `throw new Exception($"No support activies for registration form type {registrationFormVariant}");`